impl/src/bazel.rs

```diff
@@ -505,6 +505,19 @@ impl BuildRenderer for BazelRenderer {
       contents: rendered_bzl_fetch_file,
     });
 
+    // Add all lockfiles for the the binary dependencies
```

Comment:
1. (unrelated) I feel it's awkward that this file is in `bazel.rs`
2. Anytime `raze` runs with a binary dependency defined, it's going to clone that dependency to determine if anything needs to be updated. However, this may not be desirable in cases where someone reruns `raze` on the same revison months later (where transitive dependencies might have been updated). Writing out and reusing the lockfiles means users can appropriately pin their binary dependencies' transitive dependencies so subsequent runs don't generate new files. Does this answer your question? I can update the comment and the description of this pull request if so.

Reply:
Most important to me is to capture (2) tersely in the code.

(1) indicates that the separation between the abstractions has broken down (not your fault, probably my fault for not making them clearer)

Reply:
I think I've addressed this in my last commit.